boards/nxp/imx91_evk/doc/index.rst

```diff
@@ -0,0 +1,129 @@
+.. zephyr:board:: imx91_evk
+
+Overview
+********
+
+The i.MX 91 Evaluation Kit (MCIMX91-EVK board) is a platform designed
+to display the most commonly used features of the i.MX 91 applications
+processor. The MCIMX91-EVK board is an entry-level development board
+with a small and low-cost package. The board can be used by developers
+to get familiar with the processor before investing a large amount of
+resources in more specific designs.
+
+The i.MX 91 applications processor features an Arm Cortex-A55 core
+that can operate at speeds of up to 1.4 GHz.
+
+- Board features:
+
+  - RAM: 2GB LPDDR4
+  - Storage:
+
+    - SanDisk 16GB eMMC5.1
+    - microSD Socket
+  - Wireless:
+
+    - Murata Type-2EL (SDIO+UART+SPI) module. It is based on NXP IW612 SoC,
+      which supports dual-band (2.4 GHz /5 GHz) 1x1 Wi-Fi 6, Bluetooth 5.2,
+      and 802.15.4
+  - USB:
+
+    - Two USB 2.0 Type C connectors
+  - Ethernet:
+
+    - ENET: 10/100/1000 Mbit/s RGMII Ethernet connected with external PHY
+      RTL8211
+    - ENET_QoS: 10/100/1000 Mbit/s RGMII Ethernet supporting TSN connected
+      with external PHY RTL8211
+  - PCIe:
+
+    - One M.2/NGFF Key E mini card 75-pin connector
+  - Connectors:
+
+    - 40-Pin Dual Row Header
+  - LEDs:
+
+    - 1x Power status LED
+    - 2x UART LED
+  - Debug:
+
+    - JTAG 20-pin connector
+    - MicroUSB for UART debug
+
+
+Supported Features
+==================
+
+The Zephyr imx91_evk board Cortex-A Core configuration supports the following
```

Comment:
\`\` around board name, or better yet use full name as supplied to west build and call it board target

Reply:
make sense, will update. thanks